### ISSUE TITLE
[CI] RBRemoveDirectAccessToVariableTransformationTest is failing 

### ISSUE THREAD

**Issue by jecisc** [guillep/pharo-core#16291]
There is currently a failing test linked to refactorings.

testClassVariable – MacOSX64.Refactoring.Transformations.Tests.RBRemoveDirectAccessToVariableTransformationTest

```st
Got addUndo: aRefactoringChange   undo push: aRefactoringChange.  undo size > self class undoSize ifTrue: [ "Remove the element at the bottom of the stack"   undo removeLast ].  redo := OrderedCollection new instead of addUndo: aRefactoringChange   undo push: aRefactoringChange.  undo size > self class undoSize ifTrue: [ undo removeFirst ].  redo := OrderedCollection new.
Stacktrace
TestFailure
Got addUndo: aRefactoringChange
    undo push: aRefactoringChange.
    undo size > self class undoSize ifTrue: [ "Remove the element at the bottom of the stack"
        undo removeLast ].
    redo := OrderedCollection new instead of addUndo: aRefactoringChange
    undo push: aRefactoringChange.
    undo size > self class undoSize ifTrue: [ undo removeFirst ].
    redo := OrderedCollection new.
RBRemoveDirectAccessToVariableTransformationTest(TestAsserter)>>assert:description:resumable:
RBRemoveDirectAccessToVariableTransformationTest(TestAsserter)>>assert:description:
RBRemoveDirectAccessToVariableTransformationTest(TestAsserter)>>assert:equals:
RBRemoveDirectAccessToVariableTransformationTest>>testClassVariable
RBRemoveDirectAccessToVariableTransformationTest(TestCase)>>performTest
```

Maybe this is linked to recent changes of @carolahp ?

**Comment by carolahp** [comment]
Hi @jecisc , you are right, by modifying `addUndo:` I broke the test. 
I updated the test so it uses the updated method's body.
I am creating the PR where I will ask for your review

**Comment by carolahp** [comment]
Here's the [link to the PR](https://github.com/pharo-project/pharo/pull/16293)

**Comment by balsa-sarenac** [comment]
Fixed with: #16293
